### PR TITLE
Blog post and listings for tokenizers

### LINKED ISSUE #251: tokenizers pkg blog post
@lmullen placeholder for your post, close when merged

### PR DESCRIPTION
**Pull request by lmullen**
Fixes #251.

### CODE REVIEW

**Comment by sckott** [comment]
thanks for the  PR @lmullen looking over

File: _posts/2016-08-18-tokenizers-joins-ropensci.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    url: http://lincolnmullen.com
+tags:
+- R
+- natural-language
+- text-analysis
+- package
+---
+
+The R package ecosystem for natural language processing has been
+flourishing in recent days. R packages for text analysis have usuaally
```
Comment (sckott) [inline _posts/2016-08-18-tokenizers-joins-ropensci.md]: s/usuaally/usually

File: _posts/2016-08-18-tokenizers-joins-ropensci.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+[NLP](https://cran.r-project.org/package=NLP/) or
+[tm](https://cran.r-project.org/package=tm/) packages. Many of them
+depend on Java. But recently there have been a number of new packages
+for text analysis in R, most notably
+[text2vec](https://github.com/dselivanov/text2vec),
+[quanteda](https://github.com/kbenoit/quanteda), and
+[tidytext](https://github.com/juliasilge/tidytext). These packages are
+built on top of [Rcpp](http://www.rcpp.org/) instead of
+[rJava](https://cran.r-project.org/package=rJava/), which makes them
+much more reliable. And instead of the classes based on NLP, which I
```
Comment (sckott) [inline _posts/2016-08-18-tokenizers-joins-ropensci.md]: doesn't being built on Rcpp make them more portable across OSes as well?

**Comment by sckott** [comment]
@lmullen just a few small things in commit comments

**Comment by sckott** [comment]
I think we'll get this up on Monday so it catches more attention than late on a friday

**Comment by jeroenooms** [comment]
Would you mind mentioning the ropensci [hunspell](https://cran.r-project.org/web/packages/hunspell/index.html) package? The `hunspell_parse` can tokenize `text`, `html`, `latex`, `man` and `xml` in various languages and alphabets (not just english).

**Comment by lmullen** [comment]
Made the few changes you mentioned; mentioned hunspell. Thanks for looking over this, @sckott.

**Comment by sckott** [comment]
LGTM

**Pull request merged**